src/query/query_dump.c

```diff
@@ -3274,6 +3293,21 @@ qdump_print_stats_text (FILE * fp, xasl_node * xasl_p, int indent)
   qdump_print_stats_text (fp, xasl_p->scan_ptr, indent);
   qdump_print_stats_text (fp, xasl_p->connect_by_ptr, indent);
 
+  if (xasl_p->sq_cache && XASL_IS_FLAGED (xasl_p, XASL_SQ_CACHE))
+    {
+      fprintf (fp, "%*c", indent, ' ');
+      if (xasl_p->sq_cache->enabled)
+	{
+	  fprintf (fp, "SUBQUERY_CACHE (hit: %d, miss: %d, size: %lu B, status: enabled)\n",
```

Comment:
I think it is better to have no space before the letter "B".
And "%lu" should be changed to "%d".

Reply:
Okay.